xray/plot/plot.py

```diff
@@ -452,6 +452,8 @@ def newplotfunc(darray, ax=None, xincrease=None, yincrease=None,
             # pcolormesh
             kwargs['extend'] = cmap_params['extend']
             kwargs['levels'] = cmap_params['levels']
+            if 'colors' in kwargs:
```

Comment:
Looking at the wrapped matplotlib functions only `contour, contourf` have the `colors` arg in the docstring. But all 4 take a `cmap` argument.

@shoyer are you suggesting that they all have a `colors` arg? The more uniform we can make everything here, the better IMHO.